### PR TITLE
Completed html-forms exercise

### PR DESCRIPTION
**Pull request by SnowWhiteDad**
html-forms - OlaSh2
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+</head>
+
+<body>
+  <header>
+    <h1>Coding Form!</h1>
+  </header>
+  <form>
+    <div>
+      Email
+      <br>
```
Comment (Zhiwutian) [inline html-forms/index.html:15]: make sure to remove any `br` tags from this HTML page, you want to use `div`s to force input onto another line

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <header>
+    <h1>Coding Form!</h1>
+  </header>
+  <form>
+    <div>
+      Email
```
Comment (Zhiwutian) [inline html-forms/index.html:14]: make sure each input has a label with matching `for` and `id` attributes

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      <select name="ycool">
+        <option value="option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <p>Favourite Language (choose one)</p>
+    <div>
+      <input type="radio" name="favlanguage" value="html"/> HTML
```
Comment (Zhiwutian) [inline html-forms/index.html:29]: each of these inputs needs a label, you can wrap then inputs with a label if desired

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
its coming together! make sure to remove all `br` tags, and to make sure all inputs have an associated label

**Comment by SnowWhiteDad** [comment]
close

**Pull request closed without merging**